libs/log/tm_logger_test.go

```diff
@@ -66,6 +66,29 @@ func TestDebug(t *testing.T) {
 	}
 }
 
+func TestWarn(t *testing.T) {
+	var bufWarn bytes.Buffer
+
+	ld := log.NewTMLogger(&bufWarn)
+	ld.Debug("Client initialized with old header (trusted is more recent)",
```

Comment:
[blocking] how does this test Warn if it's using `Debug`? Should this be `Warn`?

```suggestion
	ld.Warn("Client initialized with old header (trusted is more recent)",
```